java-rmi/test/com/baeldung/rmi/JavaRMITest.java

```diff
@@ -0,0 +1,44 @@
+package com.baeldung.rmi;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import java.rmi.NotBoundException;
+import java.rmi.RemoteException;
+import java.rmi.registry.LocateRegistry;
+import java.rmi.registry.Registry;
+
+import org.junit.Test;
+
+public class JavaRMITest {
```

Comment:
This needs to be called JavaRMIIntegrationTest. XXXIntegrationTest is the naming convention we use for integration tests.

Reply:
Will do this.